### PR TITLE
Coinify Limits

### PR DESCRIPTION
**Pull request by jstettner**

### CODE REVIEW

**Comment by jstettner** [comment]
@plondon

**Comment by plondon** [comment]
Thanks! Can you remove the changes the package.json and package-lock.json?

**Comment by jstettner** [comment]
better now.

File: assets/js/directives/buy-quick-start.directive.js [hunk 1]
```diff
@@ -30,6 +30,12 @@ function buyQuickStart ($rootScope, currency, buySell, Alerts, $interval, $timeo
     scope.currencies = currency.coinifyCurrencies;
     scope.format = currency.formatCurrencyForView;
     scope.inMobileBuy = $rootScope.inMobileBuy;
+    scope.symbol = currency.conversions[scope.transaction.currency.code].symbol;
+
+    let exchange = buySell.getExchange();
+    this.exchange = exchange && exchange.profile ? exchange : {profile: {}};
+    this.exchangeCountry = exchange.profile ? exchange.profile.country : $stateParams.countryCode;
+    this.currency = exchange.profile ? exchange.profile.defaultCurrency : 'EUR';
```
Comment (plondon) [inline assets/js/directives/buy-quick-start.directive.js]: Where are these used?

Comment (jstettner) [inline assets/js/directives/buy-quick-start.directive.js]: My mistake. left them in from another train of thought.

File: assets/js/directives/buy-quick-start.directive.js [hunk 2]
```diff
@@ -113,6 +113,5 @@ function buyQuickStart ($rootScope, currency, buySell, Alerts, $interval, $timeo
     };
 
     scope.getInitialExchangeRate();
-
```
Comment (jtormey) [inline assets/js/directives/buy-quick-start.directive.js]: Should install a linter plugin for your editor, makes things much easier 😉

**Pull request closed without merging**